### DISCUSSION TITLE
Package multiple components

### DISCUSSION THREAD

**Discussion by khanh-le-otsv** [teambit/bit#6203] [Help]
Hello, first I want to say thank you for making such a great tool, bit is convenient to use! My question is that I have multiple icons in the folder `lit/icons` and I want to release all of them in one package: `lit/icons` instead of multiple ones: `lit/icons/info`, is there a way for me to do it with bit? Thank you.

**Comment by ranm8** [comment]
Thanks you! You should create a single component or track the `lit/icons` directory and expose all icons from its API (index.ts). Let me know if you have any issues
